tasks/scripts/terraform-smoke

```diff
@@ -18,8 +18,6 @@ echo "$SSH_KEY" > keys/id_rsa
 chmod 0600 keys/id_rsa
 ssh-keygen -y -f keys/id_rsa > keys/id_rsa.pub
 
-terraform init
```

Comment:
I think this line is still necessary since it synchronizes with the `.tfstate` file in the backend.